src/PyCall.jl

```diff
@@ -264,6 +270,13 @@ keys(o::PyObject) = Symbol[m[1] for m in pycall(inspect["getmembers"],
 # we skip wrapping Julia reserved words (which cannot be type members)
 const reserved = Set{ASCIIString}(["while", "if", "for", "try", "return", "break", "continue", "function", "macro", "quote", "let", "local", "global", "const", "abstract", "typealias", "type", "bitstype", "immutable", "ccall", "do", "module", "baremodule", "using", "import", "export", "importall", "pymember", "false", "true", "Tuple"])
 
+"""
+pywrap(o::PyObject) returns a wrapper `w` that is an anonymous module which provides (read) access to converted versions of o's members as w.member. 
```

Comment:
The  convention for Julia documentation is typically to put the function signature on the first line, by itself, with 4 spaces before it so that it is formatted as a code block:
```
"""
    pywrap(o::PyObject)

Return a wrapper `w` that is ...
"""